nautobot/extras/test_jobs/job_button_receiver.py

```diff
@@ -8,13 +8,13 @@
 
 class TestJobButtonReceiverSimple(JobButtonReceiver):
     def receive_job_button(self, obj):
-        logger.info("request.user: %s", self.request.user.username)
+        logger.info("user: %s", self.user.username)
```

Comment:
Huh. Why weren't these erroring out before?

Reply:
It appears there are no tests actually exercising this code. I tested a job to verify this shouldn't work:

```
(Pdb) self.user
<User: test-user>
(Pdb) self.request
<Context: {'id': '8fea3f57-273d-48bb-a5e0-31e4f7a61e75', 'retries': 0, 'is_eager': True, 'logfile': None, 'loglevel': 0, 'hostname': 'ac294c577bc0', 'callbacks': None, 'errbacks': None, 'headers': None, 'ignore_result': False, 'delivery_info': {'is_eager': True, 'exchange': None, 'routing_key': None, 'priority': None}, 'properties': {'nautobot_job_job_model_id': UUID('fc216cc7-174c-49f2-8af1-29f8ee262621'), 'nautobot_job_profile': False, 'nautobot_job_user_id': UUID('0cfeb801-abaf-421a-95b6-457466d418f9'), 'queue': 'default'}, 'args': (), 'called_directly': False, 'kwargs': {}}>
(Pdb) self.request.user
*** AttributeError: 'Context' object has no attribute 'user'
```